lib/charms/postgresql_k8s/v0/postgresql.py

```diff
@@ -173,3 +177,27 @@ def get_postgresql_version(self) -> str:
         except psycopg2.Error as e:
             logger.error(f"Failed to get PostgreSQL version: {e}")
             raise PostgreSQLGetPostgreSQLVersionError()
+
+    def update_user_password(self, username: str, password: str) -> None:
+        """Update a user password.
+
+        Args:
+            username: the user to update the password.
+            password: the new password for the user.
+
+        Raises:
+            PostgreSQLUpdateUserPasswordError if the password couldn't be changed.
+        """
+        try:
+            with self._connect_to_database() as connection, connection.cursor() as cursor:
+                cursor.execute(
+                    sql.SQL("ALTER USER {} WITH ENCRYPTED PASSWORD '" + password + "';").format(
```

Comment:
I couldn't find the answer. I just found that it was not working when I tried to use it as a `sql.Identifier`. I also saw in some examples that is was been used as another type, called `Placeholder`. It seems that SQL identifiers are names of entities inside the database, like users, tables, etc. Passwords seems to be something like attributes of the user entity, so it is handled different because of that.